apps/docs/src/app/core/component-docs/popover/examples/popover-complex-example/popover-complex-example.component.html

```diff
@@ -15,10 +15,10 @@
     <fd-popover [focusTrapped]="true" [focusAutoCapture]="true">
         <fd-popover-control>
             <button fd-button label="With Header"></button>
-        </fd-popover-control>
+        </fd-popover-control>x
         <fd-popover-body>
             <div fd-popover-body-header>
-                <div fd-bar [cozy]="true">
+                <div fd-bar [cozy]="true" tabindex="0">
```

Comment:
Why do we want the header to receive tab focus?

Reply:
@mikerodonnell89 

the focus on key arrow down press we dont get the focus to the header part the focus is directed only to the list elements and not the header

Reply:
tabIndex="0" makes the element tabbable, there is nothing in the header we want tabbable, the user can do this: 

<img width="209" alt="Screen Shot 2021-07-14 at 3 37 58 PM" src="https://user-images.githubusercontent.com/2471874/125682550-3e1b7dc6-acfd-4c11-a4cd-dbd1b45464ec.png">

Reply:
tabindex="0" makes the element tabbable, there is nothing in the header we want tabbable, the user can do this:

<img width="244" alt="Screen Shot 2021-07-14 at 4 26 43 PM" src="https://user-images.githubusercontent.com/2471874/125688318-ab09cf5a-6c86-4387-a3b7-7f1b212eba62.png">

We should instead use aria roles to get the screenreader to read the content